### PR TITLE
Added elmah.io

### PR DESCRIPTION
**Pull request by ThomasArdal**

### CODE REVIEW

File: _data/developer.yml
Comment (Carlgo11) [inline _data/developer.yml]:
```suggestion
      - totp
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by ThomasArdal** [comment]
Are you missing anything from me or do you wait with merging until you release a new version of the site?

**Comment by Carlgo11** [comment]
Two approvals are needed to merge so we’re just waiting for another maintainer to look at this PR before it can be merged.

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**
